### DISCUSSION TITLE
How to make the Cascade Select component scrollable like the dropdown component?

### DISCUSSION THREAD

**Discussion by MikeMork** [primefaces/community#953] [PrimeReact]
![Capture](https://github.com/primefaces/community/assets/13931845/a29dbbd8-73fe-48fb-83b9-323bac46c057)
Is it possible to easily add a scroller to a cascade select with many options?

**Comment by mertsincan** [comment]
@nitrogenous Can you assist with this question?

**Comment by nitrogenous** [comment]
Hey @MikeMork, I looked it up with our Lead UI and we realized we don't have the level option in our pass through feature. I have created an issue about this specific development. Until we implement levels into pass through you can simply achieve this by using custom css.

https://github.com/primefaces/primereact/issues/5812 

```
.p-cascadeselect-sublist ul:last-child {
    height: 300px;
    overflow: auto;
}
```
